src/Components/Category/Category.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Image from 'react-bootstrap/Image';
+
+import './Category.scss';
+import '../../Styles/fontello/css/fontello.css';
+
+function Category({ name, icon }) {
+  return (
+      <div className="icon-div" xs={12} md={6} lg={3} xl={3} >
+        <Image className={icon} alt="" src="" roundedCircle />
+        <p className="icon-name">{name}</p>
```

Comment:
To samo co wyżej odnośnie nazewnictwa klasy. Dodatkowo nazwa kategorii nie powinna znaleźć się w tagu `<p>` - bo p to paragraf, a tutaj mamy zwykłą nazwę, jakiś mały heading albo nawet `<span>` by w zupełności wystarczył.